build.sh

```diff
@@ -17,6 +17,15 @@ for suite in $SUITES; do
 	
 	./mkimage.sh -t $REPO:$suite --dir=$dir debootstrap --variant=minbase --keyring=/root/.gnupg/pubring.gpg --arch=armhf --include=sudo $suite $MIRROR
 	rm -rf $dir
+	
+	docker run --rm $REPO:$suite bash -c 'dpkg-query -l' > $suite
+	cp $suite $suite_$date
```

Comment:
@petrosagg $suite is the latest file and $suite_$date keeps tracking the history after each build